normandy/recipes/models.py

```diff
@@ -28,35 +31,54 @@ class Recipe(models.Model):
     def implementation(self):
         return ';'.join(action.implementation for action in self.actions.all())
 
+    def log_rejection(self, client=None, field=None, msg=None, op='!='):
```

Comment:
I originally expected this to be used more, and then it wasn't, and I didn't go back and fix it. Now I did.